zoia_lib/UI/ZOIALibrarian_sd.py

```diff
@@ -60,14 +60,16 @@ def sd_path(self, startup, width):
 
         if not startup:
             input_dir = QFileDialog.getExistingDirectory(
-                None, "Select an SD Card:", expanduser("~")
+                None, "Select an SD Card:", expanduser("~"),
+                options=QFileDialog.DontUseNativeDialog
             )
             if input_dir != "" and os.path.isdir(input_dir):
                 if "/" in input_dir and platform.system().lower() == "windows":
-                    # THIS IS NEEDED FOR WINDOWS.
+                    # This is (no longer needed) for Windows.
                     # This comes from a bug with QFileDialog returning the
                     # wrong path separator on Windows for some odd reason.
                     input_dir = input_dir.split("/")[0]
```

Comment:
If this is no longer needed we could likely get rid of all of this logic. Or at the very least we shouldn't hardcode the path separator anymore:
```suggestion
                    input_dir = input_dir.split(os.path.sep)[0]
```

Reply:
Let me double-check this one, I think the os.path.sep option caused an issue with SD cards. There's two checks for this in the SD script, and the implementation is different.